misk/src/main/kotlin/misk/web/actions/StatusAction.kt

```diff
@@ -30,7 +30,7 @@ class StatusAction @Inject internal constructor(
     val services = serviceManagerProvider.get().servicesByState().values().asList()
     val serviceStatus = services.map {
       when (it) {
-        is CoordinatedService -> it.service.javaClass.simpleName to it.state()
+        is DelegatingService -> it.service.javaClass.simpleName to it.state()
```

Comment:
@jjestrel fyi I added a new interface to work around `CoordinatedService` being internal

Reply:
@swankjesse and I spent a lot of time talking about this in a different PR. I think `DelegatingService` is better than what I tried in my other PR. LGTM